src/components/popover/popover.jsx

```diff
@@ -56,14 +72,16 @@ const Popover = forwardRef(function Popover(props, ref) {
     },
   }));
 
+  const className = useMemo(() => `${styles.container} ${styles[flush]}`, [flush]);
```

Comment:
A couple of notes:

- RE: useMemo: A tiny bit overkill tbh. Memoization should be on heavy computations IMO. 
- RE: interpolated className: I would prefer to see CSS composition. But I'm not sure what is the best approach in this case. 

Another thought: for stateful, 1-off "single-line" styling, I think it's okay to use the `styles` prop too. We have used this to pass state as a CSS variable from React components to CSS modules. 

e.g. How many icons are we going to show? Instead of making n-classes (`.one-icon`, `.two-icons`, ...). We set a var (`styles={{--num-of-icons: 3}}`) and use it in the styling rule (`padding-right: var(--num-of-icons, 1);`).

Reply:
I used the style api instead - b01eceb9

I didn't pass a variable to the css, instead I just set style directly on the element. I wasn't sure how to choose which css property to change using the variable method. I _think_ you could just set the `left` between 0 and the element width, but I think one thing this leaves us open to is maybe one day flushing top or bottom if that makes sense.

 Let me know what you think